packages/oauth/src/providers/linkedin.ts

```diff
@@ -42,16 +42,13 @@ export const linkedin = <_Auth extends Auth>(auth: _Auth, config: Config) => {
 
 	const getLinkedinUser = async (accessToken: string) => {
 		const linkedinUserProfile = await getProfile(accessToken);
-		const displayImageElement = linkedinUserProfile.profilePicture[
-			"displayImage~"
-		]?.elements
-			?.slice(-1)
-			?.pop();
 		const linkedinUser: LinkedinUser = {
-			id: linkedinUserProfile.id,
-			firstName: linkedinUserProfile.localizedFirstName,
-			lastName: linkedinUserProfile.localizedLastName,
-			profilePicture: displayImageElement?.identifiers?.pop()?.identifier
+			id: linkedinUserProfile.sub,
+			firstName: linkedinUserProfile.given_name,
+			lastName: linkedinUserProfile.family_name,
+			email: linkedinUserProfile.email,
```

Comment:
`linkedinUserProfile.email` can be undefined

```suggestion
			email: linkedinUserProfile.email ?? null,
```